resources/testdriver.js

```diff
@@ -170,6 +170,22 @@
          */
         freeze: function() {
             return window.test_driver_internal.freeze();
+        },
+
+        /**
+         * Send a sequence of actions
+         *
+         * This function sends a sequence of actions to the top level window
+         * to perform. It is modeled after the behaviour of {@link
+         * https://w3c.github.io/webdriver/#actions|WebDriver Actions Command}
+         *
+         * @param {Array} actions - an array of action objects to perform as
+         *                          described in docs
+         * @returns {Promise} fufiled after the actions are performed, or rejected in
+         *                    the cases the WebDriver command errors
+         */
+        action_sequence(actions) {
+            return window.test_driver_internal.action_sequence(actions);
```

Comment:
I believe we should do some target->coordinate translation here similar to click. However, we only need to do it for pointer actions as they are the only actions that may have target. @jgraham @gsnedders what do you think?

Reply:
I retract this suggestion. As scrolling the target into view is not quite possible as there might be different targets within one sequence and it is not possible to scrollintoview on different targets within one function call. So I changed the input of this function to only expect coordinates.